reconcile/test/test_jenkins_worker_fleets.py

```diff
@@ -0,0 +1,45 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from reconcile.jenkins_worker_fleets import (
+    act,
+    calculate_diff,
+    get_current_state,
+    get_desired_state,
+)
+from reconcile.test.fixtures import Fixtures
+from reconcile.utils.jenkins_api import JenkinsApi
+from reconcile.utils.terrascript_aws_client import TerrascriptClient as Terrascript
+
+fixture = Fixtures("jenkins_worker_fleets").get_anymarkup("api.yml")
```

Comment:
I think it would be much clearer if we had different fixture files for different types of data:

* gql queries
* jcasc configs
* ...

To have one giant blob makes things more difficult to follow.